Application/Source/Extensions/Extensions.swift

```diff
@@ -0,0 +1,11 @@
+import UIKit
+
+extension UILabel {
+
+    static func centeredLabel() -> UILabel {
+        let label = UILabel()
+        label.textAlignment = .center
+        label.numberOfLines = 0
+        return label
+    }
+}
```

Comment:
I believe we should be using a style for this instead of a declaring these layout attributes at initialization time.